src/pages/user/viewer/externalConnections.jsx

```diff
@@ -37,20 +40,19 @@ export const UserViewerExternalConnections = ({ user }) => {
     }
     return (
         <InnerContainer flex="1">
-            <InnerContainerRow nopadding nowrap>
-                <InputContainer column extramargin>
-                    <InputLabel small>Discord</InputLabel>
-                    {
-                        discordMapping ? (
-                            <InnerContainerRow>
-                                <S.DiscordAvatar src={`https://cdn.discordapp.com/avatars/${discordMapping.discord_id}/${discordMapping.avatar}.png`} />
-                                <S.AlignBottom>
-                                    {discordMapping.username}
-                                </S.AlignBottom>                                        
-                            </InnerContainerRow>
-                        ) : (<span>Nei</span>)
-                    }
-                </InputContainer>
+            <InnerContainerTitle>Eksterne tilkoblinger</InnerContainerTitle>
+            <InnerContainerRow>
+                <CardContainer>
+                    <CardContainerIcon>
+                        <CardContainerInnerIcon>
+                            <FontAwesomeIcon icon={faDiscord} />
+                        </CardContainerInnerIcon>
+                    </CardContainerIcon>
+                    <CardContainerText>
+                        <InputLabel small>Discord bruker</InputLabel>
+                        <CardContainerInnerText>{discordMapping ? <a href={"https://discordapp.com/users/" + discordMapping.discord_id}>{discordMapping.username}</a> : "Ingen tilkobling"}</CardContainerInnerText>
```

Comment:
Where is the avatar? People remember avatars like faces online, which is why it is important.

Reply:
Added it again